src/components/ControlPanel/event.js

```diff
@@ -0,0 +1,58 @@
+//Edited from components/Events/UserEvents/event.js
+//To dynamically pull data, use components/Events/UserEvents/event.js as reference
+
+import React from 'react'
+import EventAnalytics from './eventAnalytics'
+import Config from 'config'
+
+export default class EventCard extends React.Component {
+	//input: array of majors
+	translate(majors) {
```

Comment:
if this function doesn't really depend on anything in the `EventCard` class or doesn't have anything to do with this file, put it in `utils.js` (in `src/`) and import it from there.